tests/js/panelsTests.js

```diff
@@ -220,10 +204,70 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The model value should have been set correctly", expectedValue, that.model.speak);
     };
 
+    /*******************
+     * congratulations *
+     *******************/
+
+    fluid.defaults("gpii.tests.firstDiscovery.panel.congratulations", {
+        gradeNames: ["gpii.firstDiscovery.panel.congratulations", "autoInit"],
+        messageBase: {
+            "message": "<p>Congratulations!</p><p>Your preferences have been saved to your account.</p>",
+            "closeLabel": "close"
+        },
+        events: {
+            onClose: null
+        },
+        invokers: {
+            close: {
```

Comment:
Actually the "close" button was removed, so all of this in the test should be removed as well. I'll do that now.